### PR TITLE
Use latest version of Pronto

### PR DESCRIPTION
**Pull request by panjan**
Because of https://github.com/prontolabs/pronto/issues/312

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/489/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1321/ : SUCCESS in 1m 21s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1591/ : SUCCESS in 1m 09s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

File: lib/gooddata.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@
 $pmap_default_thread_count = 20 # rubocop:disable GlobalVars
 
 # GoodData Module
-module GoodData
+module     GoodData
```
Comment (yenkins) [inline lib/gooddata.rb]: #<struct Pronto::Git::Line line=#<Rugged::Diff::Line:47345042936480 {line_origin: :addition, content: "module     GoodData     \n">, patch=#<struct Pronto::Git::Patch patch=#<Rugged::Patch:47345023932800>, repo=#<Pronto::Git::Repository:0x0000561ebeaab710 @repo=#<Rugged::Repository:47345023933300 {path: "/src/.git/"}>>>, hunk=#<Rugged::Diff::Hunk:47345042936980 {header: "@@ -8,7 +8,7 @@ require 'pmap'\n", count: 8}>> warning: Style/ExtraSpacing: Unnecessary spacing detected. (rubocop)

Comment (yenkins) [inline lib/gooddata.rb]: #<struct Pronto::Git::Line line=#<Rugged::Diff::Line:47345042936480 {line_origin: :addition, content: "module     GoodData     \n">, patch=#<struct Pronto::Git::Patch patch=#<Rugged::Patch:47345023932800>, repo=#<Pronto::Git::Repository:0x0000561ebeaab710 @repo=#<Rugged::Repository:47345023933300 {path: "/src/.git/"}>>>, hunk=#<Rugged::Diff::Hunk:47345042936980 {header: "@@ -8,7 +8,7 @@ require 'pmap'\n", count: 8}>> warning: Style/TrailingWhitespace: Trailing whitespace detected. (rubocop)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/520/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1354/ : FAILURE in 2m 27s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1624/ : SUCCESS in 3m 12s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/521/ : SUCCESS in 1m 08s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1355/ : SUCCESS in 2m 33s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1625/ : SUCCESS in 1m 01s

**Pull request closed without merging**